### PR TITLE
Prefer createRoot rather than render for React18

### PR DESCRIPTION
**Pull request by kachick**
ref: https://reactjs.org/blog/2022/03/08/react-18-upgrade-guide.html#updates-to-client-rendering-apis

See the console log of https://github.com/pankona/strongest-mashimashi/pull/410
It has been hidden after this change.

### CODE REVIEW

File: webapp/src/index.tsx [hunk 1]
```diff
@@ -89,5 +90,4 @@ const copyText = (str: string) => {
   document.body.removeChild(tmp);
 };
 
-ReactDOM.render(<PhraGen />, document.getElementById("phrase"));
-ReactDOM.render(<PhraGen />, document.getElementById("phrase"));
```
Comment (kachick) [inline webapp/src/index.tsx:93]: Why rendering same twice? 🤔

Comment (pankona) [reply webapp/src/index.tsx:93]: Wonder

**Comment by pankona** [comment]
この変更に依らず、どうせデプロイ壊れているのでマージしちゃいます！

**Comment by kachick** [comment]
あざます！ 🍜

**Pull request merged**